tests/units/cli/test_config.py

```diff
@@ -300,6 +300,9 @@ def test_save_config(self):
                 }
             },
             'temp_dir': './pipelinewise-test-config/tmp',
+            'tap_id': 'mysql_sample',
```

Comment:
can you create another `test_config` and tests where the `query_tag` is not present?
This is to make sure the ppw and all component are backward compatible.